### PR TITLE
adding neural net options to python sklearn wrapper

### PR DESCRIPTION
**Pull request by gramhagen**
added nn and related options to sklearn vw model as requested in #1232

### CODE REVIEW

File: python/vowpalwabbit/sklearn_vw.py [hunk 1]
```diff
@@ -115,7 +115,12 @@ def __init__(self,
                  oaa=None,
                  ect=None,
                  csoaa=None,
-                 wap=None):
+                 wap=None,
+                 nn=None,
+                 dropout=None,
+                 inpass=None,
+                 meanfield=None,
+                 multitaks=None):
```
Comment (JohnLangford) [inline python/vowpalwabbit/sklearn_vw.py]: multitaks = misspelling?

**Comment by gramhagen** [comment]
oops =)

**Comment by JohnLangford** [comment]
@eisber Why is this failing?  https://ci.appveyor.com/project/JohnLangford/vowpal-wabbit/build/1.0.2064#L3353

**Comment by JohnLangford** [comment]
Merged, thanks :-)

**Pull request merged**